spec/factories/rides.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
```

Comment:
I might only include attributes that are required (not sure which are required and not), you can also use Faker or pass a block to get a more dynamic values if needed.